### ISSUE TITLE
Check model for overfitting 

### ISSUE THREAD

**Issue by india-kerle** [nestauk/ojd_daps_skills#24]
Develop two test sets:
- sample of n size in the same was the training set
- sample of n size based on the semantic DISSIMILARITY

then label them and run the model!

**Comment by india-kerle** [comment]
hmmmm we didn't really do this but we're not planning to do this for our evaluation - close @lizgzil ?
